diagnostic.md

```diff
@@ -4,7 +4,7 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+Header, Overall_summary, branch-infobar, file wrap
```

Comment:
these are not selectors! Selectors include a # for ids or a . for classes. Also, you have to be very precise: selectors generally do not start with capital letters, and they never include spaces.
.header
.overall_summary
.branch-infobar
.file-wrap

If you aren't precise with your selectors, you won't be able to select the elements you want.